packages/fxa-auth-db-mysql/lib/db/mysql.js

```diff
@@ -1364,6 +1364,16 @@ module.exports = function (log, error) {
       )
   }
 
+  const READ_SECURITY_EVENTS = 'CALL readSecurityEvents_1(?)'
+  MySql.prototype.getSecurityEvents = function (uid) {
```

Comment:
Going along with the above theme, the difference between `securityEvents` and `getSecurityEvents` isn't clear, let's rename this to `securityEventsByUid`.